app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,73 @@
+class CategoriesController < ApplicationController
+
+ before_action :set_category, only: [:show, :edit, :update, :destroy]
+
+
+  def index
+    @categories = Category.all
+  end
+
+ 
+  def show
+  end
+
+ 
+  def new
+    @category = Category.new
+
+  end
+
+ 
+  def edit
+  end
+
+  def create
+
+      @category = Category.new(category_params)
+
+    respond_to do |format|
+      if @category.save
+        format.html { redirect_to @category, notice: 'Category was successfully created.' }
+        format.json { render :show, status: :created, location: @category }
```

Comment:
please do not use scaffold
write all the method by own self. no not copy paste code
why this is format.json? do we need this?